### ISSUE TITLE
Several missing structural elements via screen reader

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#5573]
**Current behavior**
Several elements are 100% missing if you look at hthe structural elements of a page. 
(look at the autopilot there is one H2, one H1 and what is present in the projector)

**Wanted behavior**
The elements in the auto pilot should be structural meaningful
